Packs/cyberark_AIM/Integrations/CyberArkAIMV2/CyberArkAIMV2.yml

```diff
@@ -0,0 +1,68 @@
+category: Authentication
+commonfields:
+  id: CyberArkAIMV2
+  version: -1
+configuration:
+- defaultvalue: https://example.net:1234
+  display: Server URL and Port (e.g., https://example.net:1234)
+  name: url
+  required: true
+  type: 0
+- display: AppID as configured in AIM
+  hidden: false
+  name: app_id
+  required: false
+  type: 0
+- display: Folder to search in safe
+  hidden: false
+  name: folder
+  required: true
+  type: 0
+- display: Safe to search in
+  hidden: false
+  name: safe
+  required: true
+  type: 0
+- display: Credential names - comma-seperated list of credentials names in vault
+  hidden: false
+  name: credential_names
+  required: false
+  type: 12
+- display: Username
+  hidden: false
+  name: credentials
+  required: false
+  type: 9
+- display: Fetches credentials
+  name: isFetchCredentials
+  defaultvalue: "true"
+  type: 8
+  required: false
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: Query CyberArk Application Identity Manager for accounts and credentials.
+display: CyberArk AIM v2
+name: CyberArkAIMV2
+script:
+  commands:
+  - deprecated: false
+    description: Lists all credentials available
+    execution: false
+    name: cyberark-aim-list-credentials
+  dockerimage: demisto/py3ews:1.0.0.10664
```

Comment:
I think it will be better to use: `demisto/ntlm`. py3ews has a lot of libs you don't need. See: https://github.com/demisto/dockerfiles/blob/repository-info/demisto/ntlm/last.md